templates/admin/form-fields.tpl.php

```diff
@@ -1,39 +1,29 @@
 <?php
     $buttons = array(
-        array(
-            _x( 'Add New Form Field', 'form-fields admin', 'business-directory-plugin' ),
+        _x( 'Add New Form Field', 'form-fields admin', 'business-directory-plugin' ) =>
             wp_nonce_url(
 				admin_url( 'admin.php?page=wpbdp_admin_formfields&action=addfield' ),
 				'editfield'
             ),
-        ),
-        array(
-            _x( 'Preview Form', 'form-fields admin', 'business-directory-plugin' ),
-            esc_url(
-                admin_url( 'admin.php?page=wpbdp_admin_formfields&action=previewform' )
-            ),
-        ),
-        array(
-            _x( 'Manage Theme Tags', 'form-fields admin', 'business-directory-plugin' ),
-            esc_url(
-                admin_url( 'admin.php?page=wpbdp_admin_formfields&action=updatetags' )
-            ),
-        ),
+        __( 'Preview Form', 'business-directory-plugin' )      => admin_url( 'admin.php?page=wpbdp_admin_formfields&action=previewform' ),
```

Comment:
Should these be in the new array format for buttons?

Reply:
I have fixed the format for this